lib/perl/NethServer/Service.pm

```diff
@@ -249,8 +249,8 @@ sub adjust
     my $action = shift;
     my $errors = 0;
 
-    if($self->is_configured()) {
-	my $staticState = $self->is_owned() && $self->is_enabled();
+    if($self->is_configured() && $self->is_owned()) {
```

Comment:
It has been merged, but cannot work! it forgets to stop a service X when the nethserver-X package is removed.